pkg/controllers/node/termination/terminator/events/events.go

```diff
@@ -34,6 +34,16 @@ func EvictPod(pod *v1.Pod) events.Event {
 	}
 }
 
+func DeletePod(pod *v1.Pod) events.Event {
+	return events.Event{
+		InvolvedObject: pod,
+		Type:           v1.EventTypeNormal,
+		Reason:         "Deleted",
```

Comment:
I agree, because the Event is on a Pod that has not yet (AIUI) been deleted - and if the Pod has already been deleted, it may not make sense to consider it still involved.

Reply:
I guess the question is what is "deleted"? Is it when the deletion request was made or once the pod has been successfully deleted (i.e. terminationGracePeriodSeconds have expired and the pod object is removed). Right now this event maps to the former. Also, I don't think the suggested reason is actually accurate. We're deleting the pod **ahead** of the nodepool's termination grace period expired, not once it has been reached.

Reply:
I was anticipating the primary use case of this event to be cluster users checking the history of what happened to their pod, hence the use of past tense. I'm happy to use `Delete` or `Deleting` though, I'm definitely not picky there.

Reply:
Chatted with @jmdeal and @njtran about this in #karpenter-dev, we're going to use `Disrupted` in order to match with the naming conventions established for disrupted pod metrics.